test/e2e/pkg/manifest.go

```diff
@@ -174,37 +175,49 @@ func LoadManifest(file string) (Manifest, error) {
 // complex.
 func SortManifests(manifests []Manifest) {
 	sort.SliceStable(manifests, func(i, j int) bool {
-		left, right := manifests[i], manifests[j]
+		var (
+			left       = manifests[i]
+			right      = manifests[j]
+			leftScore  = len(left.Nodes) * 10
```

Comment:
So, basically, ten perturbations are worth a node? 🙂 

This seems fine, though I am not sure I understand the motivation for this particular scoring rubric. Would you mind adding a comment to sketch out the intent? (I don't really care about the mechanics, just trying to figure out what the aspiration is)

Reply:
Yeah, I did this pretty empirically (and mostly out of frustration, when my previous implementation didn't give super useful results.) 

I've changed it a bit more and added some comments.